transform/mattermost-analytics/models/marts/sales/marketing/fct_marketing_campaign_member.sql

```diff
@@ -0,0 +1,39 @@
+with marketing as (
```

Comment:
`marketing` directory should be under `marts`, not under `sales`.

Reply:
Updated directory.